lib/rules/require-capitalized-comments.js

```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```

Comment:
Although it is not fully documented but we can accept config declared through js format